### ISSUE TITLE
存储后的图片copy 失败

### ISSUE THREAD

**Issue by DaiyiLL** [SDWebImage/SDWebImage#3448]
### New Issue Checklist

* [ ] I have read and understood the [CONTRIBUTING guide](https://github.com/rs/SDWebImage/blob/master/.github/CONTRIBUTING.md)
* [ ] I have read the [Documentation](http://cocoadocs.org/docsets/SDWebImage/)
* [ ] I have searched for a similar issue in the [project](https://github.com/rs/SDWebImage/issues) and found none

### Issue Info

 Info                    | Value                               |
-------------------------|-------------------------------------|
 Platform Name           | e.g. ios / macos / tvos / watchos
 Platform Version        | e.g. 12.0 / 10.14.0 / 12.0 / 5.0
 SDWebImage Version      | e.g. 5.0.0 / 4.4.0
 Integration Method      | e.g. carthage / cocoapods / manually
 Xcode Version           | e.g. Xcode 11 / Xcode 10
 Repro rate              | e.g. all the time (100%) / sometimes x% / only once
 Repro with our demo prj | e.g. does it happen with our demo project?
 Demo project link       | e.g. link to a demo project that highlights the issue

### Issue Description and Steps

Please fill in the detailed description of the issue (full output of any stack trace, compiler error, ...) and the steps to reproduce the issue.

self.cache.store(image, imageData: nil, forKey: key, toDisk: true)
这里不传入imageData会使用image进行转换为Data，导致我通过key获取urlPath然后进行copy失败，因为我需要将存储的数据copy到沙盒的另外一个地方保存下来。是否可以提供一个方法进行保存能copy数据。
目前的做法是
        // 先设置内存
        self.cache.memoryCache.setObject(image, forKey: key, cost: image.sd_memoryCost)
        // 再通过子线程设置磁盘
        self.imageQueue.async {
            autoreleasepool {
                self.cache.diskCache.setData(image.pngData(), forKey: key)
                DispatchQueue.main.async {
                    completion?()
                }
            }
        }

**Comment by dreampiggy** [comment]
I can't get the main idea of your issue...
Why not use 

```swift
self.cache.store(image, imageData: nil, forKey: key, toDisk: true) {
   // cache store done
    if let srcPath = self.cache.cachePath(forKey: key) {
           self.moveDiskFile(src: srcPath, dst: dstPath)
    }
}
```
